client/app/visualizations/gridmap/utils.js

```diff
@@ -0,0 +1,59 @@
+import { extend } from 'lodash';
+import { AmbientLight, PointLight, LightingEffect } from '@deck.gl/core';
+
+
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 1.0,
+});
+
+const pointLight1 = new PointLight({
+  color: [255, 255, 255],
+  intensity: 0.8,
+  position: [-0.144528, 49.739968, 80000],
+});
+
+const pointLight2 = new PointLight({
+  color: [255, 255, 255],
+  intensity: 0.8,
+  position: [-3.807751, 54.104682, 8000],
+});
+
+export const lightingEffect = new LightingEffect({ ambientLight, pointLight1, pointLight2 });
+
+export const material = {
+  ambient: 0.64,
+  diffuse: 0.6,
+  shininess: 32,
+  specularColor: [51, 51, 51],
+};
+
+export const minZoom = 2;
+export const maxZoom = 17;
+
+export const MAPBOX_TOKEN = ""; // eslint-disable-line
```

Comment:
I think it's better to make it a visualization option. Nobody will rebuild the whole app just to change this token